packages/design-system/src/components/tree/tree.tsx

```diff
@@ -288,6 +289,10 @@ export const Tree = <Data extends { id: string }>({
         onBlur={keyboardNavigation.handleBlur}
         onKeyDown={keyboardNavigation.handleKeyDown}
         onClick={keyboardNavigation.handleClick}
+        css={{
```

Comment:
shoudln't this somehow automatically be prevented when content is wrapped in scrollable? we don't do marginRight for the right scroll either ... you don't want to have to define margin every time you wan to use scroll

Reply:
In most cases we don't need this margin. Here I added it only to improve a little bit last element selectability

Reply:
If you have nothing to select no need in margin